src/nxdoc/nuxeo-server/authentication-and-user-management/using-oauth2.md

```diff
@@ -274,6 +282,72 @@ Content-Type: application/json;charset=ISO-8859-1
     "token_type": "bearer"
 }
 
+```
+### Requesting an Access Token with a JWT
+
+```
+POST /oauth2/token HTTP/1.1
+Host: NUXEO_SERVER/nuxeo
+Content-Type: application/x-www-form-urlencoded
```

Comment:
Are those really needed? (`Host` and `Content-Type`)

Reply:
No, but I put it for informative purposes